### PR TITLE
update Compose release notes for version 2.15.0

### PR DESCRIPTION
**Pull request by glours**
Signed-off-by: Guillaume Lours <705411+glours@users.noreply.github.com>

### Proposed changes

Update release notes of Compose with version `v2.15.0`

### CODE REVIEW

File: compose/release-notes.md [hunk 1]
```diff
@@ -6,6 +6,38 @@ toc_max: 2
...
   - /release-notes/docker-compose/
 ---
+## 2.15.0
+{% include release-date.html date="2023-01-05" %}
+### Update
+- Dependencies upgrade: bump compose-go to v1.8.1
+- Dependencies upgrade: bump cli-docs-tool to 0.5.1
+
+### Bug fixes and enhancements
+* Add support of privileged attribute in `service.build` section. Pull Request [compose#10112](https://github.com/docker/compose/pull/10112){:
```
Comment (aevesdocker) [inline compose/release-notes.md:16]:
```suggestion
* Added support of the `privileged` attribute in the `service.build` section. Pull Request [compose#10112](https://github.com/docker/compose/pull/10112){:
```

File: compose/release-notes.md [hunk 2]
```diff
@@ -6,6 +6,38 @@ toc_max: 2
...
+  target="_blank" rel="noopener" class="_"}
+* Introduce --ignore-buildable to ignore buildable images on pull. Fixed [compose#8805](https://github.com/docker/compose/issues/8805){:
```
Comment (aevesdocker) [inline compose/release-notes.md:18]:
```suggestion
* Introduced `--ignore-buildable` to ignore buildable images on pull. Fixed [compose#8805](https://github.com/docker/compose/issues/8805){:
```

File: compose/release-notes.md [hunk 3]
```diff
@@ -6,6 +6,38 @@ toc_max: 2
...
+  target="_blank" rel="noopener" class="_"}
+* Introduce `--no-attach` to ignore some service output. Fixed [compose#8546](https://github.com/docker/compose/issues/8546){:
```
Comment (aevesdocker) [inline compose/release-notes.md:20]:
```suggestion
* Introduceed `--no-attach` to ignore some service outputs. Fixed [compose#8546](https://github.com/docker/compose/issues/8546){:
```

File: compose/release-notes.md [hunk 4]
```diff
@@ -6,6 +6,38 @@ toc_max: 2
...
+  target="_blank" rel="noopener" class="_"}
+* Fixed issue with `logs` when `driver:none` is set. Fixed [compose#9030](https://github.com/docker/compose/issues/9030){:
+  target="_blank" rel="noopener" class="_"}
+* Rely on dockerCLI.streams. Pull Request [compose#10082](https://github.com/docker/compose/pull/10082){:
```
Comment (aevesdocker) [inline compose/release-notes.md:24]:
```suggestion
* Compose now relies on dockerCLI.streams. Pull Request [compose#10082](https://github.com/docker/compose/pull/10082){:
```

File: compose/release-notes.md [hunk 5]
```diff
@@ -6,6 +6,38 @@ toc_max: 2
...
+  target="_blank" rel="noopener" class="_"}
+* Fixed issue with service hash that MUST exclude replicas. Fixed [compose#10077](https://github.com/docker/compose/issues/10077){:
+  target="_blank" rel="noopener" class="_"}
+* Check service names based on project, not running containers. Fixed [compose#9951](https://github.com/docker/compose/issues/9951){:
```
Comment (aevesdocker) [inline compose/release-notes.md:28]:
```suggestion
* Compose now checks service names based on project, not running containers. Fixed [compose#9951](https://github.com/docker/compose/issues/9951){:
```

File: compose/release-notes.md [hunk 6]
```diff
@@ -6,6 +6,38 @@ toc_max: 2
...
+  target="_blank" rel="noopener" class="_"}
+* Fixed security opts support (seccomp and unconfined). Fixed [compose#9505](https://github.com/docker/compose/issues/9505){:
+  target="_blank" rel="noopener" class="_"}
+* Fixed empty file when using compose config in case of smaller source files. Fixed [compose#10121](https://github.com/docker/compose/issues/10121){:
+  target="_blank" rel="noopener" class="_"}
+* Fixed issue with `--pull` not applied on `compose up`. Fixed [compose#10125](https://github.com/docker/compose/issues/10125){:
+  target="_blank" rel="noopener" class="_"}
+* Compose should ignore not only auto-removed containers but also "removal in progress" for orphan containers. Pull Request [compose#10136](https://github.com/docker/compose/pull/10136){:
+  target="_blank" rel="noopener" class="_"}
+* Limit build concurrency according to `--parallel`. Fixed [compose#9091](https://github.com/docker/compose/issues/9091){:
```
Comment (aevesdocker) [inline compose/release-notes.md:38]:
```suggestion
* Compose limits build concurrency according to `--parallel`. Fixed [compose#9091](https://github.com/docker/compose/issues/9091){:
```

**Comment by glours** [comment]
@aevesdocker if you're happy I'm happy 😉

**Pull request merged**